ikenna-portfolio/portfolio/src/main/webapp/script.js

```diff
@@ -16,13 +16,15 @@
  * Adds a random greeting to the page.
  */
 function addRandomGreeting() {
-  const greetings =
-      ['Hello world!', '¡Hola Mundo!', '你好，世界！', 'Bonjour le monde!'];
+  //const greetings = ['Hello world!', '¡Hola Mundo!', '你好，世界！', 'Bonjour le monde!'];
```

Comment:
In general, we should prefer not to check in commented code (here and below).

Reply:
done